sdk/tests/tutorials/properties/test_derived_properties.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+
```

Comment:
Run `black`  and clean up all this extra spacing.

Reply:
Formatting completed.